srcpkgs/electrum/patches/no-certifi.patch

```diff
@@ -2,28 +2,31 @@ Upstream: No
 Reason: Avoid certifi dependency
```

Comment:
Does this actually help? I'd rather add the dependency than carry this forever...

Reply:
Indeed, no reason for this now that it's available in the repository.